README.md

```diff
@@ -1,2 +1,71 @@
-# I-HEALTH-CAPSTONE-PROJECT
-This is a coming soon landing page for I-health. A data platform that allows for patient record to be easily manage and accessed anywhere, anytime
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# I-Health Landing page
+A coming soon website created to announce an upcoming project launch for I-health; A health care tech company
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo 
+The live demo is available at:
+
+[Live Demo Link](https://starmindz.github.io/I-HEALTH-CAPSTONE-PROJECT/)
+
```

Comment:
- [ ] [Required] Kindly deploy your page properly. I noticed that the images are not showing on the deployed version of your project. Kindly ensure that they are displayed properly. 👍 


![image](https://user-images.githubusercontent.com/77106962/181917181-b49de520-6aad-4a95-8543-c5dcf62f6824.png)